src/pages/docs/api/carrier.tsx

```diff
@@ -0,0 +1,14 @@
+import ApiLayout from "../../../layouts/api/api-layout";
+const OpenApiSpecification = require('@shipengine/connect-carrier-api/spec');
```

Comment:
?

Reply:
Pulling the openapi definition from each of the integration type packages